libGitWrap/Events/Private/GitEventCallbacks.cpp

```diff
@@ -213,8 +212,9 @@ namespace Git
         void CheckoutCallbacks::checkoutProgress(const char* path, size_t completed_steps, size_t total_steps, void* payload)
         {
             ICheckoutEvents* events = static_cast< ICheckoutEvents* >( payload );
+            Q_ASSERT( events );
 
-            qreal progress = 100 / total_steps * completed_steps;
+            qreal progress = total_steps ? (100 / total_steps * completed_steps) : 100;
```

Comment:
Btw: libgit2 itself stumbled upon this just a few days ago. They set 0 in their example, but I still think 100 is the correct value. I'll simply ask them and maybe change this in a new PR, as I'm about to merge and what's there pretty much looks ok to me. See here: https://github.com/ethomson/libgit2/commit/9af3c4169c51b967579042e392937acbc75da79d